src/test/java/com/google/jenkins/plugins/credentials/oauth/RemotableGoogleCredentialsIntegrationTest.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2018 CloudBees, Inc.
```

Comment:
@astroilov I would appreciate guidance regarding it. Your current checkstyle configuration requires Google to be a copyright owner. AFAICT it is no longer required after #16, right?

Reply:
I believe that to be correct, but I'll check with our opensource compliance team just in case.

Reply:
@astroilov 👍 Note that the current title in Google licenses may also need clarification with this team ("Copyright 2013 Google Inc. All Rights Reserved."). "All Rights Reserved" IMHO contradicts with Apache v.2 license which is used in the plugin, but IANAL

Reply:
Confirmed that this is fine.